consensus-types/blocks/proofs.go

```diff
@@ -172,3 +172,43 @@ func ComputeBlockBodyFieldRoots(ctx context.Context, blockBody *BeaconBlockBody)
 
 	return fieldRoots, nil
 }
+
+func ComputeBeaconBlockFieldRoots(ctx context.Context, block *BeaconBlock) ([][]byte, error) {
```

Comment:
Can you rename it to `ComputeBlockFieldRoots` (without the `Beacon` part) just to be consistent with the other function?